API/Data/DataContext.cs

```diff
@@ -16,7 +16,37 @@ public DataContext(DbContextOptions options) : base(options)
         public DbSet<Category> Category { get; set; }
```

Comment:
Wcześniej nie zwrócilem na to uwagi. To są DbSety czyli kolekcje czyli liczba mnoga nazwa a nie pojedyczna